executor/show.go

```diff
@@ -587,8 +587,13 @@ func (e *ShowExec) fetchShowIndex() error {
 				subPart = col.Length
 			}
 
+			tbCol, err := findColumnByName(tb.Cols(), col.Name.O)
```

Comment:
We can use `tb.Meta().Columns[col.Offset]` to get the `ColumnInfo`.

Reply:
Thx, I'll give it a try.